### DISCUSSION TITLE
Added Single-File Model Weights (for mapo-beta.safetensors)

### DISCUSSION THREAD

**Discussion by cpuimage** [mapo-t2i/mapo#5] [Show and tell]
Hi mapo team,

I have created a single-file weights model for the project and uploaded it to Hugging Face. You can find it here: https://huggingface.co/cpuimage/mapo-beta/sd_xl_mapo-beta.safetensors.

Thank you for creating such a great project! 

Feel free to check it out and let me know if there are any improvements or additional features you'd like to see.

```python 
from diffusers import StableDiffusionXLPipeline 
import torch

safetensors_file = "sd_xl_mapo-beta.safetensors"

pipeline = StableDiffusionXLPipeline.from_single_file(safetensors_file, torch_dtype=torch.float16).to("cuda")

prompt = "A lion with eagle wings coming out of the sea , digital Art, Greg rutkowski, Trending artstation, cinematographic, hyperrealistic"
image = pipeline(prompt=prompt, num_inference_steps=30).images[0]

```

**Comment by sayakpaul** [comment]
Thank you for your help!

**Comment by sayakpaul** [comment]
For convenience, I turned it into a discussion. Hopefully, that is okay with you.
